### PR TITLE
Readme and layout png location update.

### PR DESCRIPTION
**Pull request by yuqisun**
Related to issue #39
1. Readme:
Required (recommended) #CPUs, RAM?
How many hours for the 2x2?
4. Update the layout.png location.

### CODE REVIEW

**Review by tancheng — COMMENTED** [review]
Do we need to rename `run_linux_docker` to `run_windows_docker` as well?

**Comment by yuqisun** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ Docker
 The docker image is available
 [here](https://hub.docker.com/r/cgra/cgra-flow/tags).
 
+> **Recommended machine configuration**
+> * CPU: >= 6 cores
+> * Memory: 25~30G
+> 
+> **2x2 tiles run time**: ~ 40 hours (clk_period=1000)
```
Comment (tancheng) [inline README.md:34]: Can provide the rough hours for `clk_period=100000`? And also provide the link/location of the `clk_period` that user needs to change, i.e., `tools/OpenRoad-xxxx`.

The constraint and config files only exist in the docker but do not exist in CGRA-Flow repo. We may need to upload them and fix their path in the GUI in the next PRs.

Comment (yuqisun) [reply README.md:34]: Sure, testing the version submodules, will fill run times for each clk period once submodules works.
Yes, will add the both config files.

Comment (tancheng) [reply README.md:34]: I think 1000, 10000, 100000, the 3 configurations are good enough. Thanks!

**Pull request merged**